packages/components/src/stories/SquiggleChart/DistributionSizes.stories.tsx

```diff
@@ -9,41 +9,57 @@ export default meta;
 type Story = StoryObj<typeof meta>;
 
 const code = "mx(5 to 100, uniform(100, 180), 30, 60, 80, [5,5,0.3,0.3,0.3])";
+const multiplier = 2;
```

Comment:
Is this because of retina displays? Why didn't we need it before?
(I'm also confused why previous value was /2 and new is *2)

Ideally it should be encapsulated inside `<DistributionsChart>` or inside `useCanvas`, and I thought I [handled it there](https://github.com/quantified-uncertainty/squiggle/blob/56e8f50fda18ab115660ac30f904793fc4ba27a1/packages/components/src/lib/hooks/useCanvas.ts#L35) already. I can look into it more if there's a bug somewhere.

Reply:
Some heights were changed recently, I think in this other PR:
https://github.com/quantified-uncertainty/squiggle/pull/2909